### PR TITLE
[ADD] accounting: add Singapore localisations

### PR DESCRIPTION
**Pull request by tong-odoo**
Impacted versions:

- saas-16.4 or above

This commit improve below features:

- Singapore accounting documentation
- PayNow QR code documentation

task-3130633

### CODE REVIEW

**Review by toaa-odoo — APPROVED** [review]

File: content/applications/finance/fiscal_localizations/singapore.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+Activate QR codes
+-----------------
+
+Go to :menuselection:`Accounting --> Configuration --> Settings`. Under the :guilabel:`Customer
+Payments` section, activate the :guilabel:`QR Codes` feature.
+
+.. image:: singapore/sg-qr-code-setting.png
+   :align: center
+   :alt: "QR Codes" feature activated
```
Comment (jcs-odoo) [inline content/applications/finance/fiscal_localizations/singapore.rst:29]: not necessary, in my opinion

File: content/applications/finance/fiscal_localizations/singapore.rst [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+PayNow bank account configuration
+---------------------------------
+
+Go to :menuselection:`Contacts --> Configuration --> Bank Accounts` and select the bank account for
+which you want to activate PayNow. Set the :guilabel:`Proxy Type` and fill in the :guilabel:`Proxy
+Value` field depending on the type you chose.
+
+.. note::
```
Comment (jcs-odoo) [inline content/applications/finance/fiscal_localizations/singapore.rst:38]: important note, maybe? If not, you can group the next note with this one, in a bullet list
```suggestion
.. important::
```

File: content/applications/finance/fiscal_localizations/singapore.rst [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+   The account holder's country must be set to Singapore on its contact form.
+
+.. image:: singapore/sg-paynow-bank-setting.png
+   :align: center
```
Comment (jcs-odoo) [inline content/applications/finance/fiscal_localizations/singapore.rst:42]: you can delete all the centered alignment
```suggestion
```

**Review by jcs-odoo — CHANGES_REQUESTED** [review]
Hello,

Thanks for your contribution to the documentation! :)

I just want to double-check the version we should target for these two commits?
I see the support for HK FPS was made in 16.3, and not for the next 16.4 freeze, and I couldn't find any paynow task.

Thank you :)

**Comment by vin-odoo** [comment]
@jcs-odoo Howdy :smile: 
It does have to target master, it is linked to this bundle => https://runbot.odoo.com/runbot/bundle/master-l10n-sg-tong-16306
and this task => https://www.odoo.com/web#id=3130633&cids=5&menu_id=4720&action=333&active_id=1822&model=project.task&view_type=form

It adds pay now as a payment QR code for SG, and at the same time refactors the recent HK payment QR to move some things in the generic module that is shared with the SG one (and some others in the future).
So the 16.3 version of the doc is still valid for 16.3, and it has to change for 16.4

File: content/applications/finance/fiscal_localizations/singapore.rst [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+Add PayNow QR codes to invoices
+===============================
+
+PayNow is a payment service platform that allows customers to make instant domestic payments to
+individuals and merchants in Singapore dollars via online and mobile banking.
+
+Activate QR codes
+-----------------
+
+Go to :menuselection:`Accounting --> Configuration --> Settings`. Under the :guilabel:`Customer
```
[the remark this answers is not in the document]
Comment (tong-odoo) [reply content/applications/finance/fiscal_localizations/singapore.rst:24]: Right now i only see EMV format in APAC region. Not sure if I will encounter more

File: content/applications/finance/fiscal_localizations/hong_kong.rst [hunk 5]
```diff
@@ -33,8 +33,8 @@ FPS bank account configuration
 ------------------------------
 
 Go to :menuselection:`Contacts --> Configuration --> Bank Accounts` and select the bank account for
-which you want to activate FPS. Set the :guilabel:`FPS Type` and fill in the :guilabel:`FPS ID/Phone
-Number/Email Address` field depending on the type you chose.
+which you want to activate FPS. Set the :guilabel:`Proxy Type` and fill in the :guilabel:`Proxy
+Value` field depending on the type you chose.
```
[the remark this answers is not in the document]
Comment (tong-odoo) [reply content/applications/finance/fiscal_localizations/hong_kong.rst:37]: The update only applied to 16.4. In 16.3, it stays the same.

File: content/applications/finance/fiscal_localizations/singapore.rst [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+Configuration
+=============
+
+:ref:`Install <general/install>` the :guilabel:`Singapore - Accounting` module to get the features
+of the SG localization.
+
+.. image:: singapore/sg-module.png
+   :align: center
+   :alt: "Singapore - Accounting" localization module marked as "installed"
+
```
[the remark this answers is not in the document]
Comment (tong-odoo) [reply content/applications/finance/fiscal_localizations/singapore.rst:14]: sure.

**Pull request closed without merging**